README.md

```diff
@@ -32,6 +34,77 @@ TODO: finish readme later
 * Keep commit titles short and then explain them in the description of the commit.
 * **DO NOT PUSH TO MASTER BRANCH!** Push to a new branch (e.g username-whatyouchanged and create a PR)
 
+## Setup instructions
+So, Karen Bot is quite tightly integrated with my *private* API, which is good & bad news. Good news is that a lot of strain is taken off of Karen and put on my API, while the bad news is that someone without API access can't really run Karen that good. Fortunately, I have designed a lot of the API to have as little server side processing as possible, ensuring a ~~tedious~~ simple transition to your own API or just saving files on your computer!
+So this section will walk you through the necessary things to get Karen Bot atleast slightly functional :)

```

Comment:
Typo here, separate the "atleast"

Reply:
Ohh yes yes
I always say "atleast" not "at least" for some reason haha

Reply:
Oke, fixed in commit 6624f04